src/api/helpers/formatEntityData.js

```diff
@@ -74,6 +131,18 @@ export function getEditionGroupBasicInfo(editionGroup: object) {
 		};
 }
 
+/**
```

Comment:
The documentation is pretty good so far.
I think it can be improved and clarified by:
- being more to the point
- restrict the description to the limited scope of the function
- improve the examples


I've tried to apply the point above to this function docs, not as an imposition but to give you a template for these improvements. 
note for example that I don't talk about the API, because it is out of this function's scope. I only talk about the ORM because the function expects a certain object structure from it.
I wrote down what defaults to expect, and improved the example to show expected output; put yourself in the place of someone discovering the code for the first time: what would you want to know without having to look further or write any code?

```
/**
 * A function to extract the basic details of an Author ORM entity
 * @function
 * @param {object} author - an Author ORM entity
 * @returns {object} an object containing the basic data of an Author entity.
 * Each property defaults to null. If author in null or undefined, returns null.
 *
 * @example
 * 		const author = await orm.func.entity.getEntity(orm, 'Author', bbid, relations);
 *		getAuthorBasicInfo(author);
 *		/* => {
			"bbid": "86f86a31-7c51-49ed-af71-9523cca30265",
			"beginArea": null,
			"beginDate": "1904-03-23",
			"defaultAlias": {
				"aliasLanguage": "English",
				"name": "H. Beam Piper",
				"sortName": "Piper, H. Beam"
			},
			"disambiguation": null,
			"endArea": null,
			"endDate": "1964-11-06",
			"ended": true,
			"gender": "Male",
			"type": "Person"
		}
 */
```

I hope this makes sense to you. Don't hesitate to ask questions if you have any !